fidesctl/src/fidesctl/core/config/utils.py

```diff
@@ -0,0 +1,37 @@
+from errno import ENOENT
+from os import strerror
+from os.path import exists
+from typing import Any, Dict
+
+from click import echo
+from toml import dump, load
+
+
+def update_config_file(  # type: ignore
+    updates: Dict[str, Dict[str, Any]],
+    config_path: str = ".fides/fidesctl.toml",
```

Comment:
This makes sense to me, great catch @earmenda! @PSalant726 had mentioned adding the `config_path` to the config object as well so we could use it. I wonder if that will work to negate the need for a default. There were some edge cases we were testing that had this make sense at the time but it certainly seems like this would be an improvement :+1: